calcom/cal.com
# 🧹 Removed old root UI components.

## What does this PR do?

Removes old root UI components. 

**Environment**: Staging(main branch)

## Type of change

## How should this be tested?

There shouldn't be any functionality touched, so as long as the build succeeds, everything should be fine 🔥 Mainly deleted old code and replaced a few icons with the new import.
